src/dialog/engine.js

```diff
@@ -179,7 +179,9 @@ class DialogEngine {
    * Returns all the available functions along with their metadata
    */
   getAvailableFunctions() {
-    return _.values(this.functions).map(x => Object.assign({}, x, { fn: null }))
+    return _.values(this.functions)
+      .filter(x => !String(x.name).startWith('__'))
```

Comment:
`startWith` is actually `startsWith` so this code is probably failing

Reply:
Yep, good catch!